src/styles/components/data-table.scss

```diff
@@ -26,7 +26,7 @@
     color: $colour-sky-white;
     background-color: $colour-sea-blue;
     top: 0;
-    z-index: 2;
+    z-index: 1;
```

Comment:
I think there's some scss variables defined somewhere for `z-index`. It often gets messy so using the variables can help ensure where things are in relation to each other in the UI.

Reply:
good point. done